src/App/Resources/AppResources.resx

```diff
@@ -2161,7 +2161,7 @@ Scanning will happen automatically.</value>
   <data name="AccountLocked" xml:space="preserve">
     <value>Locked</value>
   </data>
-  <data name="AccountLoggedOut" xml:space="preserve">
+  <data name="LoggedOut" xml:space="preserve">
     <value>Logged out</value>
   </data>
```

Comment:
🤔 We need to be careful when modifying keys on the app resources because we lose the translations (until this one is done). Can't remember if this was added in this app version or a previous one to see if it can be modified or if we should just add a new key and mark this one as obsolete to be removed in a later version.

Reply:
I think this one is pretty old. I'll add another name to the new one.